ops/reth-cross-test/phases/fault_phase.py

```
@@ -0,0 +1,202 @@
+        print("Fault: Stopping tx-submitter-0 (finalizer only)...")
+        w3_seq = Web3(Web3.HTTPProvider(list(config.SEQUENCER_RPCS.values())[0], request_kwargs={"timeout": 10}))
+        height_before = w3_seq.eth.block_number
+
+        self._docker_cmd("stop", "tx-submitter-0")
+        time.sleep(30)  # let blocks accumulate without finalization
+
+        # Verify blocks still being produced (commits continue)
+        try:
+            height_during = w3_seq.eth.block_number
+            blocks_produced = height_during > height_before
+        except Exception:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)